pom.xml

```diff
@@ -74,7 +74,7 @@
     <aspectjrt.version>1.8.9</aspectjrt.version>
     <guice.version>3.0</guice.version>
     <guava.version>19.0</guava.version>
-    <kairosdb.version>1.1.3-1</kairosdb.version>
+    <kairosdb.version>1.2.0-1</kairosdb.version>
```

Comment:
Did you check that any shared transitives version match?

Reply:
Good catch. Updated Guice.